model/role.go

```diff
@@ -115,6 +108,23 @@ func init() {
 		PERMISSION_SYSCONSOLE_WRITE_SITE.Id: {
 			PERMISSION_EDIT_BRAND,
 		},
+		PERMISSION_SYSCONSOLE_WRITE_COMPLIANCE_DATA_RETENTION.Id: {
+			PERMISSION_CREATE_DATA_RETENTION_JOB,
+		},
+		PERMISSION_SYSCONSOLE_READ_COMPLIANCE_DATA_RETENTION.Id: {
+			PERMISSION_READ_JOBS,
```

Comment:
Shouldn't this be PERMISSION_READ_DATA_RETENTION_JOB and PERMISSION_READ_COMPLIANCE_EXPORT_JOB?

Reply:
Nice catch @maxerenberg